modules/scylladb/src/test/java/org/testcontainers/scylladb/ScyllaDBContainerTest.java

```diff
@@ -0,0 +1,181 @@
+package org.testcontainers.scylladb;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import org.junit.Test;
+import org.testcontainers.utility.DockerImageName;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeDefinition;
+import software.amazon.awssdk.services.dynamodb.model.BillingMode;
+import software.amazon.awssdk.services.dynamodb.model.CreateTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.KeySchemaElement;
+import software.amazon.awssdk.services.dynamodb.model.KeyType;
+import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ScyllaDBContainerTest {
+
+    private static final DockerImageName SCYLLADB_IMAGE = DockerImageName.parse("scylladb/scylla:6.2");
+
+    private static final String BASIC_QUERY = "SELECT release_version FROM system.local";
+
+    @Test
+    public void testSimple() {
+        try ( // container {
+            ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)
+            // }
+        ) {
+            scylladb.start();
+            // session {
+            CqlSession session = CqlSession
+                .builder()
+                .addContactPoint(scylladb.getContactPoint())
+                .withLocalDatacenter("datacenter1")
+                .build();
+            // }
+            ResultSet resultSet = session.execute(BASIC_QUERY);
+            assertThat(resultSet.wasApplied()).isTrue();
+            assertThat(resultSet.one().getString(0)).isNotNull();
+            assertThat(session.getMetadata().getNodes().values()).hasSize(1);
+        }
+    }
+
+    @Test
+    public void testSimpleSsl()
```

Comment:
Thanks for adding this! I think cql cli command would not work unless is configured properly. We should configure it properly when ssl is enabled https://opensource.docs.scylladb.com/stable/operating-scylla/security/gen-cqlsh-file.html

Reply:
@eddumelendez  yes, if ssl enabled on the server for client connections in order to cqlsh to work we need to configure ssl for it too, but it is not used here, in test containers, right? I didn't get your point, could you pls clarify what else you would like to see in this PR? thanks!

Reply:
Users can execute commands using via execInContainer method provided by Testcontainers API. Testcontainers should provide a proper configuration for the container to run properly. In this case, external clients will connect properly because the sdk client is configured but not inside the container. I think we should fix that.

Reply:
I've added a test example how to make cqlsh work with same ssl setup.

Reply:
Thanks! WDYT about adding `withSsl(MountableFile certificate, MountableFile keyfile, MountableFile truststore)` and copy those files as part of `ScyllaDBContainer`. If so, then create the cqlshrc.

```java
ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)
    .withConfigurationOverride("scylla-test-ssl")
    .withSsl(certificate, keyfile, truststore)
```

Reply:
@eddumelendez Sure, added withSsl. Also used `SSL_CERTFILE` env variable instead of rewriting `cqlshrc` file as there could be issues with the connection hostname specified in the file and also it simplifies setup.